stack_plan.go

```diff
@@ -32,6 +32,10 @@ type StackPlans interface {
 
 	// PlanDescription returns the plan description for a stack plan.
 	PlanDescription(ctx context.Context, stackPlanID string) (*JSONChangeDesc, error)
+
```

Comment:
Can we add a AwaitQueued? Only then the eventstream URL is set and we need that for showing progress events.

Reply:
I don't think queued is a state for stack plan. Do you mean running_queued?

Reply:
I'm not too sure what I meant anymore, just ignore this comment 🙈